### PR TITLE
Make known issues clearer in readme and issues template

### LINKED ISSUE #418: Document package management signing issues on nanoserver
https://github.com/PowerShell/PowerShell-Docker/issues/417#issuecomment-621964048

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Make known issues clearer in readme and issues template
  - Known issues have been moved to wiki for easier editting

Fixes #418 

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -4,12 +4,16 @@ For Windows PowerShell 5.1 issues, suggestions, or feature requests please use t
...
-- make sure you are able to repro it on the latest released version. 
+- make sure you are able to repro it on the latest released version.
 - Search the existing issues.
 - Fill out the following repro template:
 
 If it's not a bug, please remove the template and elaborate the issue in your own words.
 -->
+## Check list
+
+- [ ] Verified that this is not a [Known Issues](https://github.com/PowerShell/PowerShell-Docker/wiki/Known-Issues)
```
Comment (RDIL) [inline .github/ISSUE_TEMPLATE.md:15]:
```suggestion
- [ ] Verified that this is not a [Known Issue](https://github.com/PowerShell/PowerShell-Docker/wiki/Known-Issues)
```

File: README.md [hunk 2]
```diff
@@ -52,31 +52,9 @@ PS /> Write-Host "Hello, World!"
...
-
-#### PowerShell Get only works with CurrentUser Scope
-
-Due to [known issues with the nano-server-insider](https://github.com/Microsoft/Virtualization-Documentation/blob/live/virtualization/windowscontainers/quick-start/Insider-Known-Issues.md#build-16237),
-you must specify `-Scope CurrentUser` when using `Install-Module`.  Example:
-
-```powershell
-Install-Module <ModuleName> -Scope CurrentUser
-```
+[Known Issues](https://github.com/PowerShell/PowerShell-Docker/wiki/Known-Issues)
```
Comment (RDIL) [inline README.md:57]:
```suggestion
See [Known Issues](https://github.com/PowerShell/PowerShell-Docker/wiki/Known-Issues)
```

**Comment by TravisEz13** [comment]
Thanks for reviewing, @RDIL!

**Comment by RDIL** [comment]
@TravisEz13 ready for merge

**Pull request merged**